setup/auditlog/odoo/addons/auditlog

```diff
@@ -0,0 +1 @@
+../../../../auditlog
```

Comment:
Is this file should be present in commit?

Reply:
ouch, it came with "[ADD] setup.py". It won't hurt as `setup/` are update by the bot (the same that introduced that commit). I'm gonna drop it anyway :wink: